cmd/carbonapi/http/render_handler.go

```diff
@@ -269,15 +269,14 @@ func renderHandler(w http.ResponseWriter, r *http.Request) {
 		// Obtain error code from the errors
 		// In case we have only "Not Found" errors, result should be 404
 		// Otherwise it should be 500
+                returnCode = http.StatusNotFound
```

Comment:
Please run `go fmt` on the file to correct formating.

Reply:
Done.